apps/storybook/src/components/Breadcrumb.stories.tsx

```diff
@@ -99,3 +99,25 @@ export const WithDropdown: Story = {
     ),
   },
 };
+
+export const Truncated: Story = {
+  args: {
+    children: (
+      <>
+        <BreadcrumbItem>
+          <BreadcrumbLink href="/">
```

Comment:
i think we can just remove the height and display styles from `BreadcrumbLink`. there's no need for them.

Reply:
there's quite a few redundant styles there now - we can also remove `rounded` - not sure why that's there on breadcrumb link